docs/configuration.md

```diff
@@ -58,21 +62,34 @@ The `cluster-lock.json` has the following schema:
   "distributed_validators": [                               // Length equal to num_validators (n*32ETH staked).
     {
       "distributed_public_key":  "0x123..abfc",             // DV root pubkey
-      "public_shares": [ "0x123..abfc", "0x123..abfc"],     // The public share of each operator (length of num_operators)
-      "fee_recipient": "0x123..abfc"                        // Fee recipient address of this validator. Defaults to definition fee_recipient if empty.
+      "public_shares": ["0x123..abfc", "0x123..abfc"],      // The public share of each operator (length of num_operators)
+      "deposit_data": {...},                                // Deposit data to activate this validator
+      "builder_registration": {...}                         // Pre-generated signed builder registration for the validator
     }
   ],
   "lock_hash": "0xabcdef...abcedef",                        // Hash of the cluster definition and distributed validators. Uniquely identifies a cluster lock.
-  "signature_aggregate": "0xabcdef...abcedef"               // BLS aggregate signature of the lock hash signed by all the key shares of all the distributed validators. Proves that the key shares exist and attested to being part of this cluster.
+  "signature_aggregate": "0xabcdef...abcedef",               // BLS aggregate signature of the lock hash signed by all the key shares of all the distributed validators. Proves that the key shares exist and attested to being part of this cluster.
+  "node_signatures": ["0xabcdef...abcedef"]                 // Signatures of the lock hash for each operator defined in the cluster definition
```

Comment:
`// Signatures of the lock hash by each operator. Proves that this lock file (and the validators) was generated by all the operators`